lib/rules/restrict-full-import.js

```diff
@@ -34,7 +34,7 @@ module.exports = {
       return left.type === 'Identifier' ||
       (
         left.type === 'ObjectPattern' && left.properties.some((prop) => {
-          return prop.type === 'SpreadProperty' || prop.type === 'ExperimentalRestProperty';
+          return prop.type === 'SpreadProperty' || prop.type === 'ExperimentalRestProperty' || prop.type === 'RestElement';
```

Comment:
> If you have written a custom rule that relies on rest operators having the ExperimentalRestProperty type, you should update it to also work with rest operators that have the RestElement type.

This fixed failing tests as per the migration guide suggestion [here](https://eslint.org/docs/user-guide/migrating-to-5.0.0#-when-using-the-default-parser-rest-operators-now-have-type-restelement)